pkg/infra/filestorage/wrapper.go

```diff
@@ -114,25 +114,35 @@ func (b wrapper) removeRoot(path string) string {
 	return Join(Delimiter, strings.TrimPrefix(path, b.rootFolder))
 }
 
-func (b wrapper) Get(ctx context.Context, path string) (*File, error) {
+func (b wrapper) getOptionsWithDefaults(options *GetFileOptions) *GetFileOptions {
+	if options == nil {
+		return &GetFileOptions{WithContents: true}
+	}
+
+	return options
+}
+
+func (b wrapper) Get(ctx context.Context, path string, options *GetFileOptions) (*File, bool, error) {
 	if err := b.validatePath(path); err != nil {
-		return nil, err
+		return nil, false, err
```

Comment:
What does the `bool` in the return type for get represent? Whether or not the request was successful / unsuccessful?

Reply:
as discussed during daily, the flag indicates whether the file was found. I added it to improve the developer experience (`if found` rather than `if file != nil`) on a suggestion from @FZambia